web/src/app/browser/build.sh

```diff
@@ -41,6 +41,11 @@ builder_describe_outputs \
 
 #### Build action definitions ####
 
+do_clean() {
+  rm -rf "$KEYMAN_ROOT/web/build/$SUBPROJECT_NAME"
+  rm -rf "$KEYMAN_ROOT/web/build/publish"
```

Comment:
>  while two of the `app/` projects output to a publish folder to prepare for releases, their clean steps did not actually delete any previously-existing entries.

I see app/ has: browser, ui, webview.
Should this line be in a clean action at /web/src/app/ or /web?

Reply:
There is no `/web/src/app/build.sh`, while `/web` almost entirely delegates to all of the child scripts - and thus to `/web/src/app/browser/build.sh` and `/web/src/app/ui/build.sh`.